src/locales/en_US.ts

```diff
@@ -255,7 +255,7 @@ const strings = {
   wallet_list_add_token: 'Add Token',
   wallet_list_referral_link_currency_invalid: 'Currency to create is invalid',
   wallet_list_referral_link_currency_loading: 'Hang tight. Creating wallet necessary for this promotion',
-  wallet_list_referral_link_ask_wallet_creation: 'You need %s wallet for this promotion, do you want to create one?',
+  wallet_list_referral_link_ask_wallet_creation_1s: 'You need a %1$s wallet for this promotion. Would you like to create one?',
```

Comment:
Seems better with this wording. 
Might as well fix the key, too, since it needs a re-translation.